roles/check_reboot/vars/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+# vars file for check_reboot
+webhook_token: "{{ lookup('ansible.builtin.env', 'SLACK_WEBHOOK_TOKEN') }}"
```

Comment:
Can you include an example of the token in the comments?  Does it include the https://* or not?